Gemfile

```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 
 ruby File.read(".ruby-version").strip
 
-gem 'rails', '~> 5.2.2.1'
+gem 'bootsnap', '>= 1.1.0', require: false
```

Comment:
Is this the equivalent of spring?
Does Rails need this?

Reply:
Yeah, have a read of the upgrade notes in the description above. Don't neeeeed it but its part of Rails 5.2 and apparently helps.